web/views/skill_view.ex

```diff
@@ -5,5 +5,5 @@ defmodule CodeCorps.SkillView do
   attributes [:title, :description, :inserted_at, :updated_at]
 
   has_many :role_skills, serializer: CodeCorps.RoleSkillView
-  has_many :roles, serializer: CodeCorps.RoleView
+  has_many :user_skills, serializer: CodeCorps.UserSkillView
```

Comment:
Do we really want all the `user_skills` across the app loaded for each `Skill`? I wouldn't think so.

I can see adding it for a `User`, but not for a `Skill`.